pkg/cache/snapshot.go

```diff
@@ -177,3 +198,17 @@ func (c *ClusterQueue) accumulateResources(cohort *Cohort) {
 		}
 	}
 }
+
+// GetRequestableQuota recalculates cohort.RequestableResources when LendingLimit enabled,
+// because when borrowing from the cohort, need to count cq's own nominal quota in.
+func (c *ClusterQueue) GetRequestableQuota(fName kueue.ResourceFlavorReference, rName corev1.ResourceName) (val int64) {
```

Comment:
```suggestion
// RequestableQuota recalculates cohort.RequestableResources when LendingLimit enabled,
// because when borrowing from the cohort, need to count cq's own nominal quota in.
func (c *ClusterQueue) RequestableQuota(fName kueue.ResourceFlavorReference, rName corev1.ResourceName) (val int64) {
```

https://go.dev/doc/effective_go#Getters

Reply:
That makes sense. Done.